custom/icds_reports/utils/aggregation_helpers/monolith/awc_infrastructure.py

```diff
@@ -13,11 +13,12 @@ class AwcInfrastructureAggregationHelper(BaseICDSAggregationHelper):
     column_names = (
         'supervisor_id',
         'timeend',
-        'awc_building', 'source_drinking_water', 'toilet_functional',
+        'awc_building', 'source_drinking_water', 'toilet_facility', 'type_toilet', 'toilet_functional',
         'electricity_awc', 'adequate_space_pse',
         'adult_scale_available', 'baby_scale_available', 'flat_scale_available',
         'adult_scale_usable', 'baby_scale_usable', 'cooking_utensils_usable',
         'infantometer_usable', 'medicine_kits_usable', 'stadiometer_usable',
+        'preschool_kit_available', 'preschool_kit_usable'
```

Comment:
Hey, no specific reason. Though order does not technically affect the result because of the use of sub query. But, are you mostly concern about readability and understandability or there are other concerns as well?

Reply:
That's fine. Was just wondering if there was a reason. I think it is nice/more readable if similar lists are in similar orders